res/shaders/deferred_lighting.hlsl

```diff
@@ -1,4 +1,5 @@
 #include "lighting_calculations.hlsl"
+#include "../../src/SHADING_DEFINES.h"
```

Comment:
Unfortunately this is not going to work in a packaged build, because it has a completely different folder structure. You can see it for yourself by downloading the build artifact

I don't see any other way to make it work, other than making two files with the same content, one for shaders and one for src, which is very meh. Alternatively we could change the build folder structure but this will involve a lot of fuckery with symlinks

Reply:
Yup, I was wondering about that. I guess we are going to need two copies :(